src/design/Layout.scss

```diff
@@ -339,3 +339,18 @@ svg:not(:root) {
 .continue-button-placement {
     margin-top: 30px !important;
 }
+
+.time-restrictions-button-placement[class] {
```

Comment:
We should be able to use the js-enabled class to hide the button when js is disabled:

```
.time-restrictions-button-placement[class] {
    display: none;
}

.js-enabled {
    .time-restrictions-button-placement[class] {
        display: block;
        margin-top: auto;
        margin-bottom: 0px;
        max-height: 40px;
        min-width: 125px;
    }
}
```